src/math/abs.jl

```diff
@@ -51,9 +54,12 @@ abs_min([-3,1,2]) # returns 1
 abs_min([-7,-3,6]) #returns -3
 ```
 
-Contributed By:- [Ashwani Rathee](https://github.com/ashwani-rathee)
+Contributed By:- [Ashwani Rathee](https://github.com/ashwani-rathee) and [Rratic](https://github.com/Rratic)
 """
-function abs_min(x)
-    result = sort(x, by=abs; alg=QuickSort)
-    return result[1]
+function abs_min(x::Vector{N})where N<:Number
```

Comment:
I guess `Number` can be changed to `Real` becuase comparing `Complex` does not make sense

Reply:
Unless there is a conflict with another `abs_min` method, there is no need to provide types